js/search.js

```diff
@@ -1,40 +1,35 @@
 /**
  * Common search features
  */
-define([
-    'jquery',
-    'logger',
-    'search/temporal-coverage-facet',
-    'widgets/range-picker',
-], function($, log) {
-    'use strict';
+import $ from 'jquery';
+import log from 'logger';
+import 'search/temporal-coverage-facet';
+import 'widgets/range-picker';
 
-    $(function() {
-        log.debug('search started');
+$(function() {
+    log.debug('search started');
 
-        // Display toolbar depending on active tab
-        $('.search-tabs a[data-toggle="tab"]').on('shown.bs.tab', function (e) {
-            var $tab = $(e.target);
-            $('.btn-toolbar.wrapper').each(function() {
-                var $this = $(this);
-                if ($this.data('tab') == $tab.attr('href')) {
-                    $this.removeClass('hide');
-                } else {
-                    $this.addClass('hide');
-                }
-            });
-        });
-
-        $('.advanced-search-panel .list-group-more').on('shown.bs.collapse', function () {
-            $('button[data-target="#' + this.id + '"]').hide();
+    // Display toolbar depending on active tab
+    $('.search-tabs a[data-toggle="tab"]').on('shown.bs.tab', function(e) {
```

Comment:
I wish I could: this rely on bootstrap js which is emiting a jQuery `shown.bs.tab` event.
I think relying on vanilla/ES6 instead of Bootstrap might need a dedicated PR.